### ISSUE TITLE
Is it possible to switch between preferred words when "group related words" option enabled and switch audio between grouped words?

### ISSUE THREAD

**Issue by Triramama** [FooSoft/yomichan#2263]
I have enabled this option so entries of slightly differently spelled or pronounced words would get put together in a single search column, for example it is very useful for cases where i create my anki cards and alternative pronunciation/kanji versions of same expression are logged together in the card. And overall it is a very handy feature that lets me know of all the variations of the word i looked up without having to scroll around for them manually. 
![image](https://user-images.githubusercontent.com/97356540/198065144-234103fa-253b-4119-a2e6-69cfcf234dfe.png)
However what i cannot do is make yomichan create my anki card with automatically inserted audio from the word that isn't highlighted in blue (i would assume there should be a way to switch the highlight but it doesn't seem to be possible), thus as you can see on the picture i can't create audio with さっきゅう only with the former そうきゅう  without turning off the option and viewing さっきゅう as a separate search. I would really like if somebody could be done to amend this little Inconvenience, ideally i would like a way to create a card with more than one audio track, in such cases where there's alternative pronunciations for the word, but for starters making yomichan create an anki card with audio from one of the unhighlighted words when "group related words" option enabled would be great. thanks

**Comment by toasted-nutbread** [comment]
You're correct that there is currently no way to do this. The audio download code is rather messy due to the limitations of then handlebars implementation, so the current function to acquire audio is only able to get it from one text source, which it usually assumes is the first.

As for the "highlighted" blue text, this is actually unrelated to the selection - the colors in this context actually indicate things like term rarity (which is not indicated in other display modes).
